src/index.js

```diff
@@ -92,7 +90,7 @@ function syncReduxAndRouter(history, store, selectRouterState = SELECT_STATE) {
 
   const unsubscribeHistory = history.listen(location => {
     const route = {
-      path: history.createPath(location),
+      path: history.createHref(location),
```

Comment:
This is the line that is breaking the tests in the browser. Changing back to `createPath` makes the tests pass.